RawWaterManagement/OpenChannelJunction/examples/example-normalized.jsonld

```diff
@@ -0,0 +1,113 @@
+      "addressCountry": "Fast into brother see his keep discover. Different individual film sense.",
+      "postalCode": "Place go occur people south. Third site material particularly. Evidence industry red tend focus fly mind these.",
+      "postOfficeBoxNumber": "She everybody actually more true. East play decide before style. He fear body thank effect above.",
+      "areaServed": "Dream newspaper foreign coach attorney true economy. Skin film natural mission."
+    }
+  },
+  "areaServed": {
+    "type": "Property",
+    "value": "Floor nearly stand always right. Ready information skin oil boy."
+  },
+  "id": "urn:ngsi-ld:dataModel:id:PYGQ:66276695",
+  "dateCreated": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "1978-08-03T08:57:03Z"
+    }
+  },
+  "dateModified": {
+    "type": "Property",
+    "value": {
+      "@type": "DateTime",
+      "@value": "2021-03-23T00:16:00Z"
+    }
+  },
+  "source": {
+    "type": "Property",
+    "value": "Rate concern party level food."
+  },
+  "name": {
+    "type": "Property",
+    "value": "Situation fast ok including report answer capital note. Decide eat learn. Price kitchen why would."
+  },
+  "alternateName": {
+    "type": "Property",
+    "value": "Home customer data can after. Degree too value into some wall whom."
+  },
+  "description": {
+    "type": "Property",
+    "value": "Able dog national whose measure consider source. Quality democratic peace actually remember study focus. Myself court bring young others assume business."
+  },
+  "dataProvider": {
+    "type": "Property",
+    "value": "They agency impact my. Apply movement fast ten without serious long."
+  },
+  "owner": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:WNCG:70269691",
+      "urn:ngsi-ld:dataModel:items:INOD:84827103"
+    ]
+  },
+  "seeAlso": {
+    "type": "Property",
+    "value": [
+      "urn:ngsi-ld:dataModel:items:NHOU:08830312",
+      "urn:ngsi-ld:dataModel:items:ULRL:23255556"
+    ]
+  },
+  "type": "OpenChannelJunction",
+  "position": {
+    "type": "Property",
+    "value": {
+      "distance": 864.6,
+      "refPoint": "urn:ngsi-ld:dataModel:refPoint:JXFD:60487647"
+    }
+  },
+  "downstreamNode": {
+    "type": "object",
+    "value": "urn:ngsi-ld:dataModel:downstreamNode:CBWI:21948924"
+  },
+  "upstreamNode": {
+    "type": "object",
+    "value": "urn:ngsi-ld:dataModel:upstreamNode:MWGU:81565938"
+  },
+  "observedBy": {
+    "type": "object",
+    "value": "urn:ngsi-ld:dataModel:observedBy:GIWE:80160975"
+  },
+  "uniqueName": {
+    "type": "Property",
+    "value": "Draw whose sort language ball. Themselves your majority feeling fact by. Sport music white owner onto knowledge."
+  },
+  "tag": {
+    "type": "Property",
+    "value": "First drug contain start almost wonder. Live bed serious theory type. Together type music hospital. Every speech support time operation wear often."
+  },
+  "waterOutflow": {
```

Comment:
Use of unitcode is encouraged to ensure consistency of collected data. Otherwise, no one knows if e.g. a temperature is expressed in °C or K 
The specification recommends to take it from UN CEFACT (https://unece.org/DAM/cefact/recommendations/rec20/rec20.zip)